src/cloud/project/services-elastic.md

```diff
@@ -199,5 +199,24 @@ elasticsearch:
 If you use the ElasticSuite third-party plugin, you must [update the `{{site.data.var.ct}}` package]({{ site.baseurl }}/cloud/project/ece-tools-update.html) to version 2002.0.19 or later.
 When setting up ElasticSuite, add the configuration settings to the `ELASTICSUITE_CONFIGURATION` deploy variable. This configuration saves the settings across deployments.
 
+### Remove plugins for Elasticsearch
+Removing the plugin entries from `elasticsearch:` in `.magento/services.yaml` does not uninstall or disable them as you might expect. You must take the additional step of reindexing your Elasticsearch data. This is intentional to prevent possible loss or corruption of data that depends on these plugins.
+
+**To remove Elasticsearch plugins:**
+
+1. Remove the Elasticsearch plugin entries from your `.magento/services.yaml` file.
+1. Commit the `.magento/services.yaml` changes to your Cloud repo.
```

Comment:
You might consider adding the explicit steps to commit changes since there are several. Here's the standard step we use for that:

1. Add, commit, and push your code changes.

   ```bash
   git add -A
   ```

   ```bash
   git commit -m "Remove Elasticsearch plugin"
   ```

   ```bash
   git push origin <branch-name>
   ```